fix: update readme TDE-173 (#588)
linz/topo-processor: README.md

## Description

The Topo Processor is a collection of small components that can be combined together to create a pipeline. It can be run on a local workstation or using AWS Batch.

These components include transforming data into cloud optimised formats like [COG](https://www.cogeo.org/) and the creation of [STAC](http://stacspec.org/) metadata.

## Installation

> ### Requirements ro run Topo Processor locally:

Reviewer: ```suggestion
### Requirements to run Topo Processor locally:
```

The global user configuration is defined by environment variables, example environment variables are found in the `.env` file.

> ### Requirements ro run Topo Processor using AWS Batch:

Reviewer: ```suggestion
### Requirements to run Topo Processor using AWS Batch:
```

#### Software

```shell
yarn

yarn build
```

#### AWS Batch Stack deployment

**_NOTE:_** [AWS deployment is done automatically through GitHub Actions.](#aws-deployment--ci--cd)

To deploy the Batch via CDK locally:

On the AWS account you are logged into

```shell
yarn build

npx cdk deploy
```

### AWS Roles

> To allow the system to perform cross account AWS request, you'll need to config AWS roles inside of a AWS SSM parameter.

Reviewer: ```suggestion
To allow the system to perform cross account AWS requests, you'll need to config AWS roles inside of an AWS SSM parameter.
```
